packages/form-builder/src/blocks/fields/settings.js

```diff
@@ -1,6 +1,58 @@
 import {InspectorControls} from "@wordpress/block-editor";
 import {PanelBody, PanelRow, TextControl, ToggleControl} from "@wordpress/components";
 import {__} from "@wordpress/i18n";
+import {noop} from "lodash";
+
+const Edit = function (props) {
+
+    const {
+        attributes: {label, placeholder, isRequired, options},
+        setAttributes,
+        onLabelTextControlBlur = noop
+    } = props;
```

Comment:
Let's make sure to destruct the props inline within the component function so they're more explicit.

Reply:
Resolved 8778f1ed